app/src/main/java/de/danoeh/antennapod/activity/MediaplayerActivity.java

```diff
@@ -652,6 +654,7 @@ void setupGUI() {
         setContentView(getContentViewResourceId());
         sbPosition = findViewById(R.id.sbPosition);
         txtvPosition = findViewById(R.id.txtvPosition);
+        seekDisplay = findViewById(R.id.seek_display);
```

Comment:
Have you checked a video podcast? I think this line will crash because you only added the view to the layout of the audio player. Would be good to also add it to the video player layout.